src/Renderer/ThreeExtented/FeatureToolBox.js

```diff
@@ -0,0 +1,559 @@
+/**
+ * Generated On: 2016-09-28
+ * Class: FeatureToolBox
+ * Description:
+ */
+
+import THREE 					from 'THREE';
+import CVML 					from 'Core/Math/CVML';
```

Comment:
In fact, we have only one solution to do extrusion with threejs through ExtrudeGeometry, but as i said before, we will lost the style for the 3D building and make dificulties for StreetView when we project texture on the building. THREE.ExtrudeGeometry does not fit in our case. If you dont want to have this module, we can install tri2poly or earcut nodejs library.